src/core/tsi/s2a/handshaker/s2a_handshaker_client.cc

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "src/core/tsi/s2a/handshaker/s2a_handshaker_client.h"
+
+namespace grpc {
+namespace experimental {
+
+tsi_result s2a_handshaker_client_start_client(
+    const s2a_handshaker_client* client) {
+  return TSI_UNIMPLEMENTED;
+}
+
+tsi_result s2a_handshaker_client_start_server(
+    const s2a_handshaker_client* client, grpc_slice* bytes_received) {
+  return TSI_UNIMPLEMENTED;
```

Comment:
So I guess you will have a separate PR that does the implementation right?

Reply:
Yes, that's correct. There is not much content to this one, just agreeing on the API's.